src/problem1.js

```diff
@@ -1,6 +1,38 @@
 function problem1(pobi, crong) {
-  var answer;
-  return answer;
+  if (!isValidPage(pobi) || !isValidPage(crong)) {
+    return -1;
+  }
+  else {
+    pobiScore = Math.max(getMaxScore(pobi[0]), getMaxScore(pobi[1]));
+    crongScore = Math.max(getMaxScore(crong[0]), getMaxScore(crong[1]));
+    return getWinner(pobiScore, crongScore);
+  }
+}
+
+function getMaxScore(page) {
+  var digits = page.toString().split('').map(Number);
+  pageSum = 0;
+  pageMul = 1;
+
+  digits.forEach(digit => {
+    pageSum += digit;
+    pageMul *= digit;
+  });
+
+  return Math.max(pageSum, pageMul);
+}
+
+function getWinner(scoreA, scoreB) {
+  if (scoreA > scoreB)  return 1;
+  else if (scoreA < scoreB) return 2;
+  else if (scoreA == scoreB)  return 0;
+  else return -1;
```

Comment:
위 3개의 조건들이 각각 `>, <, ==` 이니깐

이 조건들이 모두 아닌 경우는 없지 않나요?

(= 제일 마지막 `else` 조건은 절대 못들어오는거 아닌가용?)

```suggestion
  if (scoreA > scoreB)  return 1;
  else if (scoreA < scoreB) return 2;
  else return 0;  // scoreA == scoreB
```